include/mdbfakeglib.h

```diff
@@ -39,6 +39,7 @@ typedef const void * gconstpointer;
 typedef uint8_t guint8;
 typedef guint32 GQuark;
 typedef guint32 gunichar;
+typedef signed int gssize;
```

Comment:
I believe this should be a `signed long`: https://people.gnome.org/~ryanl/glib-docs/glib-Basic-Types.html#gssize

Reply:
Looks like I was looking at the wrong manual page here: http://irtfweb.ifa.hawaii.edu/SoftwareDocs/gtk20/glib/glib-basic-types.html#gssize.

What the command line argument to force the build to use the fake glib code?

Reply:
Never mind, I found the --disable-glib option in the configure script.